.github/workflows/pullrequests.yml

```diff
@@ -56,16 +47,22 @@ jobs:
         python -m pip freeze
     - name: Run Tests
       run: |
-        python -m pytest
+        python -m pytest -v -ra --junitxml=pytest.xml
+    - name: Publish Test Report
+      uses: mikepenz/action-junit-report@v2
+      if: always()
+      with:
+        report_paths: pytest.xml
+
```

Comment:
Why the new action to publish the junit report?

Reply:
So that you don't have to scroll through the test output to view what had failed, and instead it shows it like a test report in the actions output.

Reply:
All I'm seeing is a flood of `JUnit Test Report` actions which do not seem at all more understandable or parsable. What am I missing?

Reply:
That is because there are no failures, but here is the test report https://github.com/plangrid/flask-rebar/pull/286/checks?check_run_id=13404513126

they will show you what the failures are if there are any

here is an example
https://github.com/gtmanfred/teststack/runs/12086499726

Reply:
Thanks for the examples.
Is there any way to configure it so that the output show what combination was used on that test run? As it stands I personally just find that view confusing as it's not clear to me which entries in the test matrix experienced which failures.

Reply:
I don't think you can directly tie the output to the job that ran, but all it is doing is dumping the output into the job summary, and from there you can see which job had failed tests, and that can be used to take you to the full log.
https://github.com/plangrid/flask-rebar/actions/runs/4951665623

Reply:
<img width="919" alt="Screenshot 2023-05-11 at 3 20 00 PM" src="https://github.com/plangrid/flask-rebar/assets/732321/85525672-742a-4066-b087-a386e9127856">
You would do that by clicking "View job logs"

Reply:
Thanks, appreciate all the answers. I'm sold 👍